plugins/contrib/gen.go

```diff
@@ -0,0 +1,19 @@
+//go:build ignore && goprobe_contrib
```

Comment:
See above, generation only triggers in generator mode _and_ if `goprobe_contrib` is set. Refer to https://github.com/els0r/goProbe/pull/216#discussion_r1335207351 for my reasoning about using `goprobe_contrib` as build tag name.